lib/selectivity/rspec.rb

```diff
@@ -36,6 +36,14 @@ def selectivity_unselect(value, *args)
 
       private
 
+      def find_selectivity_input(from, options)
+        find(:div, from, options)
+      rescue Capybara::ElementNotFound
+        label = find('label', {text: from}.merge(options))
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.